src/Controller/ModulesController.php

```diff
@@ -109,11 +109,12 @@ public function view($id) : ?Response
             return $this->redirect(['_name' => 'modules:list', 'object_type' => $this->objectType]);
         }
 
-        $object = $response['data'];
         $revision = Hash::get($response, 'meta.schema.' . $this->objectType . '.revision', null);
         $schema = $this->Schema->getSchema($this->objectType, $revision);
 
-        $this->set(compact('object', 'schema'));
+        $object = $response['data'];
+        $included = (!empty($response['included'])) ? $response['included'][0] : null;
```

Comment:
it's probably safer to use `$response['included']` complete array, even though it will usually be just one element

Reply:
When we have more than one element, should we show multiple streams/thumbs? it makes sense